2.4/test/run

```diff
@@ -169,32 +166,17 @@ function run_self_cert_test() {
   run "diff ./configcert ./servercert >cert.diff"
 }
 
-function run_all_tests() {
-  for test_case in $TEST_LIST; do
-    : "Running test $test_case"
-    $test_case
-  done;
-}
-
 function run_dockerfiles_test() {
-  run "ct_test_app_dockerfile examples/Dockerfile 'https://github.com/sclorg/httpd-ex.git' 'Welcome to your static httpd application on OpenShift' app-src" 0
-  run "ct_test_app_dockerfile examples/Dockerfile.s2i 'https://github.com/sclorg/httpd-ex.git' 'Welcome to your static httpd application on OpenShift' app-src" 0
+  run "ct_test_app_dockerfile ../examples/Dockerfile 'https://github.com/sclorg/httpd-ex.git' 'Welcome to your static httpd application on OpenShift' app-src" 0
```

Comment:
Are you sure about this? What about adding a symlink if it is missing?

Reply:
I did not notice the symlink. Thanks for catching this. It is fixed in the new commit.